package.json

```diff
@@ -23,7 +23,7 @@
         "replace": "replace",
         "tslint:check": "node_modules/.bin/tslint -c tslint.json \"src/**/*.ts\" -e \"**/node_modules/**\"",
         "tslint:fix": "node_modules/.bin/tslint -c tslint.json --fix \"src/**/*.ts\" -e \"**/node_modules/**\"",
-        "webpack": "webpack",
+        "webpack": "webpack --progress --colors",
```

Comment:
What do you think about this? https://github.com/FormidableLabs/webpack-dashboard Not for this PR but wanted to note here so that you get a chance to take a look at that.

Reply:
Noted, I put it on the list for dev happiness here: https://github.com/vmware/clarity/issues/1624